csrc/scheduler/resize.cpp

```diff
@@ -419,6 +491,10 @@ void ResizeScheduler::schedule(Fusion* fusion, const HeuristicParams* params) {
   // [..., I0/bdimx/max_gdimx, max_gdimx(BIDx), bdimx(TIDx), vec_factor] or
   // [..., I0/bdimx(BIDx), bdimx(TIDx), vec_factor]
 
+  // Before propagating the reference transformations, make sure all
+  // tensors are ready to get transformations from the ref
+  prepareForBackwardTransformPropagation(ref_tv);
```

Comment:
> // This may not work if there's a tensor that is reshaped
  // from the reference tensor, but that should not be the case as the
  // reference is picked by the same routine used for the pointwise
  // scheduler.

nitpick, the comment in the new line 500 - 503 reads a bit strange in the context of this PR.
Sounds like it is indeed happening and we are patching that now. Should we remove that comment?